src/collections/posts/were-looking-for-authors.md

```diff
@@ -0,0 +1,46 @@
+---
+title: We're looking for Authors!
```

Comment:
Perhaps title-casing this? The Join our Author Team is also title-cased.

Reply:
Good point for consistency! Will do that